### PR TITLE
service.py: SysV command generator fix

### PR DESCRIPTION
**Pull request by svirt**
For SysV case command generator is fixed in sys_v_init_command_generator()

### CODE REVIEW

**Comment by svirt** [comment]
I hope this fix can be merged. Thanks

File: virttest/staging/service.py [hunk 1]
```diff
@@ -290,7 +290,6 @@ def sys_v_init_command_generator(command):
         command_name = "chkconfig"
         command = "off"
     elif command == "raw_status":
-        command_name = "chkconfig"
```
Comment (yangdongsheng) [inline virttest/staging/service.py]: Okey, looks good. Did you test it on a sysV env?

**Comment by svirt** [comment]
yes I did some tests, and besides now I am working on adding this support to the rest of managedsave tests

**Comment by yangdongsheng** [comment]
Okey, I think we should let it in now. @Hao-Liu what's you opinion? If that's fine to you, please consider merging it. Thanx. :)

**Comment by Hao-Liu** [comment]
Of cause `chkconfig` don't have `status` subcommand. ACK this and merging. Thanks @svirt @yangdongsheng.

**Comment by yangdongsheng** [comment]
BTW, @Hao-Liu , Long time no see. I have been working in a Filesystem and driver team from last year, then I have no much time for our autotest/virt-test now, sorry about it :(. But I think I will still do some review work for us.  How is going on?  Is there some cool stuff in our community recently?

**Comment by yangdongsheng** [comment]
Thanx @Hao-Liu , Great to know there are some new guys interested in our project. About Yu, he went to another city and there should be a period he need for his new job and life, maybe a new girl, LOL. I believe that would not be long before his coming back.

Well, thanx for your information and I will start to help to do some work on PR reviewing, maybe start from next week.

Thanx a lot.

**Comment by yumingfei** [comment]
Sorry, everybody, I still need several weeks to be stable. :(
And glad to see so many new attenders in this project, I expect and believe you can make enormous enhancement to libvirt tests. :)

**Pull request merged**